android/lib/build.gradle

```diff
@@ -16,6 +16,9 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+        debug {
+            multiDexEnabled true
```

Comment:
Is this required..?

Reply:
Yes, required for instrumentation testing

Reply:
But this enables it not only for testing... Why is it required for instrumentation testing?